### PR TITLE
[Platformer] Fix the key condition mapping.

### PR DESCRIPTION
**Pull request by D8H**
shadow00dev reported the issue.

I did a copy paste and didn't realize that the attributes weren't in the same order. I should have written a test case for it.

### CODE REVIEW

File: Extensions/PlatformBehavior/tests/JumpAndFallingPlatformer.spec.js [hunk 1]
```diff
@@ -1313,5 +1313,22 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
...
+      'Jump',
+      'Right',
+      'Left',
+      'Up',
+      'Down',
+      'Ladder',
+      'Release',
+      'Release Ladder',
+    ].forEach((key) => {
+      it.only(`can tell that ${key} key is used`, function () {
```
Comment (AlexandreSi) [inline Extensions/PlatformBehavior/tests/JumpAndFallingPlatformer.spec.js:1327]: 💜

**Review by AlexandreSi — APPROVED** [review]

File: Extensions/PlatformBehavior/tests/JumpAndFallingPlatformer.spec.js (hunk 1 above)
Comment (4ian) [inline Extensions/PlatformBehavior/tests/JumpAndFallingPlatformer.spec.js:1327]: it.only to be removed

File: Extensions/PlatformBehavior/tests/JumpAndFallingPlatformer.spec.js [hunk 2]
```diff
@@ -1313,5 +1313,22 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
...
+        object.getBehavior('PlatformerObject').simulateControl(key);
+        runtimeScene.renderAndStep(1000 / 60);
+        object.getBehavior('PlatformerObject').isUsingControl(key);
```
Comment (4ian) [inline Extensions/PlatformBehavior/tests/JumpAndFallingPlatformer.spec.js:1330]: expect(...) missing here

**Review by 4ian — APPROVED** [review]
👍

**Pull request merged**